src/Spout/Writer/Style/Style.php

```diff
@@ -302,6 +338,9 @@ public function mergeWith($baseStyle)
         if (!$this->hasSetWrapText && $baseStyle->shouldWrapText()) {
             $mergedStyle->setShouldWrapText();
         }
+        if (!$this->hasSetBackgroundColor && $baseStyle->shouldApplyBackgroundColor()) {
```

Comment:
should you add a check && $baseStyle->getBackgroundColor() !== self::DEFAULT_BACKGROUND_COLOR?
No need to set the background color if it's the default one.